Modix.Services/Moderation/ModerationService.cs

```diff
@@ -70,7 +70,7 @@ public interface IModerationService
         /// </summary>
         /// <param name="infractionId">The <see cref="InfractionEntity.Id"/> value of the infraction to be rescinded.</param>
         /// <returns>A <see cref="Task"/> which will complete when the operation has completed.</returns>
-        Task RescindInfractionAsync(long infractionId);
```

Comment:
Would you mind updating the XML documentation for this method with a brief description of the new parameter?

Reply:
No problem! Embarrassed I missed that myself